src/main/kotlin/com/github/warningimhack3r/intellijshadcnplugin/backend/sources/Source.kt

```diff
@@ -111,6 +111,8 @@ abstract class Source<C : Config>(val project: Project, private val serializer:
             .split("\n").joinToString("\n") { line ->
                 // Remove // comments
                 line.substringBefore("//").trim()
+                // Remove /* */ comments
+                line.replace(Regex("/\\*.*?\\*/"), "").trim()
```

Comment:
This sounds like a good idea but it causes issues
See [29e4726](https://github.com/WarningImHack3r/intellij-shadcn-plugin/commit/29e472674143797da1354f30e97e01023ad48d8e) and #66

Reply:
Alright, I'll look into a more robust solution for this tomorrow